### ISSUE TITLE
Term for every item type

### ISSUE THREAD

**Issue by denismaier** [citation-style-language/csl-evolution#17]
as per https://github.com/citation-style-language/zotero-bits/issues/70

**Comment by denismaier** [comment]
Open question: Shall this involve a more sophisticated mechanism, as per @bwiernik's suggestion:

===
Could this set of terms get special treatment so that it can be called with something like term="item-type" or term="type" and then have the correct term generated based on the item type? This would make coding things like fallback rules for APA or MLA much shorter.
===

**Comment by bwiernik** [comment]
@cormacrelf [mentioned](https://github.com/citation-style-language/zotero-bits/issues/70#issuecomment-445091423) it shouldn't be difficult to implement this:

> Nah, implementation is easy, just a single extra hash lookup. Go for it.
>
> (Yes, I’m working on a processor.)

**Comment by denismaier** [comment]
So I'd suggest we aim for the maximalist option. Good for everyone? We can of course do this in two steps. First terms, lookup logic second.

**Comment by bwiernik** [comment]
So 1.1 terms, 2.0 logic?

**Comment by denismaier** [comment]
Better yet 1.1 terms, 1.2. logic.

**Comment by denismaier** [comment]
@bdarcus Why label "documentation"?

**Comment by bdarcus** [comment]
You mean because it's redundant?

Mainly just so have it easily scanned and filtered.

Thoughts?

>

**Comment by bwiernik** [comment]
I'm not sure what this label is supposed to mean.

**Comment by bdarcus** [comment]
Sorry; tired. I mixed this up with another issue.

Fixed.

**Comment by adam3smith** [comment]
I think we have consensus for yes here?
Let's definitely do 1.1 terms and try to get feedback from other implementers on whether the logic is indeed small enough for 1.2 or should wait until 2.0

**Comment by denismaier** [comment]
I'll move to accepted.

**Comment by bwiernik** [comment]
I wasn't aware/thinking of there being a 1.0.2, so we should clarify the scope of these three version numbers so we are all on the same page.

**Comment by bdarcus** [comment]
We need a minor version bump for these string changes.

I posted elsewhere the suggestion for 1.0.2 and 1.1 for now, and 2.0 is
bigger changes.

**Comment by bwiernik** [comment]
What's the distinction between 1.0.2 and 1.1?

**Comment by adam3smith** [comment]
> What's the distinction between 1.0.2 and 1.1?

Yeah -- I think that's worth clarifying over on discourse. When I talked about versioning in the past, I didn't think 1.0.2 would exist (Rintze and I considered 1.0.1 probably a misnomer)

**Comment by bdarcus** [comment]
How about here:

https://discourse.citationstyles.org/t/versioning-of-csl-and-the-csl-spec/841
